pkg/config/config.go

```diff
@@ -98,18 +98,21 @@ func validateConfig(config Config) error {
 		return fmt.Errorf("you must set heartbeat.interval")
 	}
 
-	if config.Conference.HeartbeatConfig.Timeout > 30 {
-		return fmt.Errorf("heartbeat.timeout must be 30s or lower")
+	// MSC3898 dictates what are the limits for these values.
+	if config.Conference.HeartbeatConfig.Timeout < 30 {
+		return fmt.Errorf("heartbeat.timeout must be 30s or higher")
 	}
 	if config.Conference.HeartbeatConfig.Interval > 30 {
 		return fmt.Errorf("heartbeat.interval must be 30s or lower")
 	}
 
-	if config.Conference.HeartbeatConfig.Timeout < 5 {
-		return fmt.Errorf("heartbeat.timeout must be 5s or higher")
+	// While MSC3898 isn't explicit about this, we don't want users to shoot
+	// themselves in the foot.
+	if config.Conference.HeartbeatConfig.Timeout > 60*2 {
+		return fmt.Errorf("it is not recommended for heartbeat.timeout to be higher than 2m")
```

Comment:
We are checking the `Timeout` value above in the code, maybe it would be better to just combine the `if` condition and return an error instead with a text that "timeout is expected to be in range A..B"?

Reply:
Well, from my POV they are different errors: one is about what the MSC allows and the other is about what we recommend

Reply:
I think it depends whether one is an ignorable warning & the other is an error - if so then they'd want to be separate. If they're both hard errors I'd just make them the same. I still think just hard coding the values might prevent overthinking this. :)